front/src/App/model/talk.js

```diff
@@ -29,4 +40,12 @@ export default class Talk {
   isIn(talks) {
     return talks.some((talk) => talk.id === this.id);
   }
+
+  colorForTalkManagement() {
+    return this.track
+      ? this.track.color
+      : this.isAssigned()
+      ? 'status-ok'
+      : `accent-${this.isToSchedule() ? 3 : this.isInqueue() ? 2 : 4}`;
```

Comment:
por favor no encadenemos ternarios. Son dificiles de leer. Pueden tener sentido al hacerlos pero despues es un esfuerzo cognitivo para uno en el futuro y para otros devs.

Reply:
Mismo estos ternarios:

`${this.isToSchedule() ? 3 : this.isInqueue() ? 2 : 4}`

Se vuelve medio inentendible

Reply:
Mismo estos ternarios:

`${this.isToSchedule() ? 3 : this.isInqueue() ? 2 : 4}`

Se vuelve medio inentendible